pylint/checkers/utils.py

```diff
@@ -512,7 +512,7 @@ def parse_format_string(
     format_string: str,
 ) -> Tuple[Set[str], int, Dict[str, str], List[str]]:
     """Parses a format string, returning a tuple of (keys, num_args), where keys
-    is the set of mapping keys in the format string, and num_args is the number
+    are the set of mapping keys in the format string, and num_args is the number
```

Comment:
NBD but I would have left this as is, `keys` is the variable name.

Reply:
Thanks, you're right ! 
3e28cb1